style.css

```diff
@@ -82,3 +87,331 @@
   padding-left: 0;
   padding-top: 20px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  width: 375px;
+  height: 2788px;
+  background: #f7f7f9;
+}
+
+.cardworks1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+  height: 670px;
+  width: 350px;
+}
+
+.snapshot {
+  width: 340px;
+  height: 220px;
+}
+
+.leftblock {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 340px;
+  height: 288px;
+}
+
+.prim-txt {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-content: space-between;
+  padding: 0;
+  width: 295px;
+  height: 75px;
+}
+
+.prim-txt h2 {
+  width: 295px;
+  height: 36px;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.prim-txt img {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  width: 236px;
+  height: 32px;
+}
+
+.leftblock p {
+  margin-top: 40px;
+  width: 220px;
+  height: 50px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tag ul {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+  padding-top: 0;
+  width: 203px;
+  height: 24px;
+  justify-content: space-between;
+}
+
+.html {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 53px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.css {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 45px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.js {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 89px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  text-align: center;
+}
+
+.action {
+  margin-top: 0;
+  padding-top: 0;
+  width: 295px;
+  height: 72px;
+}
+
+.button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 100px;
+  height: 25px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  text-align: center;
+}
+
```

Comment:
- [ ] Kindly implement styles for the two other states (`:hover` and `:active`) of a button for all your buttons as provided by the design:

![image](https://user-images.githubusercontent.com/99715640/219578545-13558b00-5c93-42ba-bff8-e51a76319666.png)

_**Kindly check the figma template for the correct style properties**_ :muscle: You gat this! :hugs: